### PR TITLE
fix: build with Xcode 12.4 SQPIT-781

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-781" title="SQPIT-781" target="_blank">TaskSQPIT-781</a>  Build frameworks compiles on both Xcode 12 and 13
  </td></table>
  <br />
 

# What's new in this PR?

ref: https://github.com/wireapp/wire-ios-protos/pull/60

## dependency
- [ ] https://github.com/wireapp/wire-ios-shared-resources/pull/70

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/billypchan-patch-2 # TODO: UNDo Branch to fetch the jobs template from
```
Comment (johnxnguyen) [inline azure-pipelines.yml:6]: Todo?

Comment (KaterinaWire) [inline azure-pipelines.yml:6]: Don't forget to revert it

**Review by KaterinaWire — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (billypchan) [inline azure-pipelines.yml:6]:
```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**236**. Will create a **patch** release after merged.
Next version is 84.0.4.

**Comment by billypchan** [comment]
release minor

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release minor** for PR #**236**. Will create a **minor** release after merged.
Next version is 84.1.0.

**Pull request merged**